frontend/src/app/testQueue/TestTimer.tsx

```diff
@@ -240,8 +242,10 @@ export const TestTimerWidget = ({ timer, context }: Props) => {
         data-testid="timer"
         aria-label="Start timer"
```

Comment:
I agree this should be added on the aria label, although I think we should create a separate ticket for it since it'll also affect the existing queue item component which will still be used in prod until the new test card feature flag is enabled